### ISSUE TITLE
%loss with tenth of a percentage point

### ISSUE THREAD

**Issue by O-ring** [schweikert/fping#427]
Hello everyone,

I would like to ask if it is possible to report the percentage of lost packets with a tenth of a percentage point. For example, the current output is as follows:

xmt/rcv/%loss = 86390/85564/0%

I think it would be useful to have an output like this:

xmt/rcv/%loss = 86390/85564/0,9%

or even better:

xmt/rcv/%loss = 86390/85564/0,95%

Thanks,
Marco

**Comment by schweikert** [comment]
Unfortunately, we can't easily do this because it would break backwards compatibility. I recommend calculating the percentages on your own if you need better precision.
